graphitron-codegen-parent/graphitron-java-codegen/src/main/java/no/sikt/graphitron/mappings/TableReflection.java

```diff
@@ -399,6 +399,13 @@ public static Optional<Class<?>> getFieldType(String table, String name) {
         return getField(table, name).map(Typed::getType);
     }
 
+    public static boolean fieldTypeIsCLOB(String table, String name) {
+        return getField(table, name)
+                .flatMap(it -> Optional.ofNullable(it.getDataType().getSQLDataType()))
+                .map(it -> it.getName().equalsIgnoreCase("clob"))
```

Comment:
```suggestion
                .map(it -> it.equals(SQLDataType.CLOB))
```
?
Ikke så viktig så du kan la det være hvis du vil